repository/src/test/java/org/alfresco/repo/action/ActionServiceImplTest.java

```diff
@@ -1295,6 +1294,89 @@ public void testExecutionTrackingDetails() {
        assertEquals(123455, action.getExecutionEndDate().getTime());
        assertEquals(null, action.getExecutionFailureMessage());
     }
+
+    @Test
+    public void testActionExposureBasedOnConfiguration()
+    {
+        globalConfig.remove("org.alfresco.repo.action.public-test-action.exposed");
+        globalConfig.remove("org.alfresco.repo.action.test.public-test-action.exposed");

```

Comment:
I got a bit confused at first between `public-test-action.exposed` and `test.public-test-action.exposed`. On reading through the test, it does become clear, but I had to look carefully to avoid mistaking one for the other.